modules/_labs/select/react/package.json

```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "@emotion/core": "^10.0.28",
+    "@popperjs/core": "^2.1.1",
```

Comment:
`@popperjs/core` can be removed as a dependency if we export `Rect` from the CK Popper.

Reply:
Removed `@popperjs/core` as a dependency now that `Rect` is exported from the CK Popper.